build.yaml

```diff
@@ -750,6 +750,26 @@ steps:
    dependsOn:
     - default_ns
     - hail_repl_image
+ - kind: runImage
+   name: upload_test_resources_to_gcs
+   image:
+     valueFrom: base_image.image
+   script: |
+     rm -rf repo
+     mkdir repo
+     cd repo
+     {{ code.checkout_script }}
+     gcloud -q auth activate-service-account --key-file=/test-gsa-key/key.json
+     gsutil -m cp -r hail/src/test/resources/* gs://7-day/{{ token }}/test/resources/
```

Comment:
What's this for?

Reply:
The Hail Python tests rely on text files, among other things, in `src/test/resources`.